caffe2/CMakeLists.txt

```diff
@@ -701,11 +702,19 @@ IF (USE_ROCM)
 ELSEIF(USE_CUDA)
   set(CUDA_LINK_LIBRARIES_KEYWORD PRIVATE)
   cuda_add_library(torch_cuda ${Caffe2_GPU_SRCS})
+  cuda_add_library(__torch_cuda_sp ${Caffe2_GPU_SRCS_SP} STATIC)
+  add_dependencies(__torch_cuda_sp Caffe2_PROTO)
+  if (INTERN_BUILD_ATEN_OPS)
+  add_dependencies(__torch_cuda_sp aten_op_header_gen)
```

Comment:
Nit: spaces

Reply:
I understand the indent feedback, but what are the preferred formatting in `CMakeLists.txt`?
I.e.: `if (foo)` vs `if(foo)` vs `IF(foo)` vs `IF (foo)`. My personal preference is former (similar to C), but it doesn't seem that any particular style is followed in CMakeLists.txt

Reply:
Actually, pushed commit to the same branch that removes spaces

Reply:
I generally just match the file style, but let me see what you came up with.